www/src/models/mineralSite/Reference.ts

```diff
@@ -42,7 +43,7 @@ export class Reference {
       document: this.document.serialize(),
       comment: this.comment,
       property: this.property,
-      // pageInfo: this.pageInfo
+      pageInfo: this.pageInfo
```

Comment:
you need to fix deserialize/serialize because of pageInfo -- convert between camelcase/snakecase